impl/pom.xml

```diff
@@ -70,51 +45,124 @@
             <distribution>repo</distribution>
         </license>
     </licenses>
-    <issueManagement>
-        <system>github</system>
-        <url>https://github.com/eclipse-ee4j/jsp-api/issues</url>
-    </issueManagement>
+
+    <developers>
+        <developer>
+            <id>yaminikb</id>
+            <name>Yamini K B</name>
+            <organization>Oracle Corporation</organization>
+            <organizationUrl>http://www.oracle.com</organizationUrl>
+        </developer>
+    </developers>
+    <contributors>
+        <contributor>
+            <name>Kin Man Chung</name>
+        </contributor>
+    </contributors>
+
     <mailingLists>
         <mailingList>
-          <name>JSTL dev mailing list</name>
+          <name>Jakarta Server Pages dev mailing list</name>
           <post>jsp-dev@eclipse.org</post>
           <subscribe>https://dev.eclipse.org/mailman/listinfo/jsp-dev</subscribe>
           <unsubscribe>https://dev.eclipse.org/mailman/listinfo/jsp-dev</unsubscribe>
           <archive>https://dev.eclipse.org/mhonarc/lists/jsp-dev</archive>
         </mailingList>
-    </mailingLists> 
+    </mailingLists>
+
     <scm>
         <connection>scm:git:https://github.com/eclipse-ee4j/jsp-api.git</connection>
         <developerConnection>scm:git:ssh://git@github.com/eclipse-ee4j/jsp-api.git</developerConnection>
         <url>https://github.com/eclipse-ee4j/jsp-api</url>
         <tag>HEAD</tag>
     </scm>
+    <issueManagement>
+        <system>github</system>
+        <url>https://github.com/eclipse-ee4j/jsp-api/issues</url>
+    </issueManagement>
+
+    <properties>
+        <spec.version>3.0</spec.version>
+        <extensionName>jakarta.servlet.jsp</extensionName>
+        <bundle.symbolicName>org.glassfish.web.jakarta.servlet.jsp</bundle.symbolicName>
+        <vendorName>Eclipse Foundation</vendorName>
+    </properties>
+
+    <dependencies>
+        <!-- Jakarta EE Dependencies -->
+        <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>5.0.0-M1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.el</groupId>
+            <artifactId>jakarta.el-api</artifactId>
+            <version>4.0.0-RC1</version>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.servlet.jsp</groupId>
+            <artifactId>jakarta.servlet.jsp-api</artifactId>
+            <version>3.0.0-M1</version>
+        </dependency>
+        
+        <!-- Other dependencies -->
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.10.7</version>
+        </dependency>
+        <dependency>
+          <groupId>eclipse</groupId>
+          <artifactId>jdtcore</artifactId>
```

Comment:
Odd indentation here.

I think we should just settle on 2 spaces, no tabs and be done with it.
That's the default on both Eclipse IDE and IntelliJ for XML anyway.

Reply:
I think the default for Jakarta is 4 spaces, indeed no tabs. I'll update it, thanks again!